src/appleseed/renderer/kernel/lighting/backwardlightsampler.cpp

```diff
@@ -82,11 +92,23 @@ BackwardLightSampler::BackwardLightSampler(const Scene& scene, const ParamArray&
         m_non_physical_lights_cdf.prepare();
     
     // Build the light tree.
-    m_light_tree_light_count = m_light_tree.build(
-        m_light_tree_lights,
-        m_emitting_triangles);
+    if (m_use_light_tree)
```

Comment:
Use curly braces for multiline bodies even if it's a single statement:

    if (m_use_light_tree)
    {
        m_light_tree_light_count =
            m_light_tree.build(
                m_light_tree_lights,
                m_emitting_triangles);
    }

But honestly I'd write it on a single line, it's not overly long:

    if (m_use_light_tree)
        m_light_tree_light_count = m_light_tree.build(m_light_tree_lights, m_emitting_triangles);